impl/sql/mutationstorage/mutation_logs.go

```diff
@@ -163,6 +164,7 @@ func (m *Mutations) send(ctx context.Context, ts time.Time, directoryID string,
 // equal to batchSize items greater than start.
 func (m *Mutations) HighWatermark(ctx context.Context, directoryID string, logID int64,
 	start time.Time, batchSize int32) (int32, time.Time, error) {
+	startQuery := start.Add(quantum - 1).Truncate(quantum)
```

Comment:
Please update PR description. It says "Define ... test", but in reality the more notable point here is the impl is changing.